Source/Core/Request.js

```diff
@@ -148,6 +150,36 @@ define([
         this.cancelled = true;
     };
 
+    /**
+     * Duplicates a Request instance.
+     *
+     * @param {Request} [result] The object onto which to store the result.
+     *
+     * @returns {Request} The modified result parameter or a new Resource instance if one was not provided.
+     */
+    Request.prototype.clone = function(result) {
```

Comment:
Looks like you have a comment in `Resource.clone` about purposefully not cloning the `request`, so is this still used or should this file be reverted?